test/manual/cardano-node/DBMigration.md

```diff
@@ -0,0 +1,53 @@
+# DB Migration from previously released version
+
+## OS
+
+Windows, MacOS, Linux
+
+**strongly recommended to test on all platforms**
+
+## Start previous version of the wallet
+
+1. Get **previous** release -> https://github.com/input-output-hk/cardano-wallet/releases
+2. Start cardano-node and cardano-wallet on `testnet` using latest [config](https://hydra.iohk.io/build/4547830/download/1/index.html). Make sure both are fully synced.
+
+```bash
+cardano-node run \
+	--config ./*-config.json \
+  --topology ./*-topology.json \
+	--database-path ./db \
+	--socket-path ./node.socket
+
+cardano-wallet serve --port 8090 \
+  --node-socket ../relay1/node.socket \
+  --testnet testnet-byron-genesis.json  \
+  --database ./wallet-db
+```
+
+3. Produce some data in the cardano-wallet DB e.g.
```

Comment:
Can we add commands for all of these? If you don't use the cli every day it might take some time to figure out all the correct commands.

Reply:
Sure!

On related note I recommend https://github.com/piotr-iohk/ikar. The best thing for _manual testing_ of cardano-wallet that's currently on the market. ;)

Reply:
Addressed in 2c40953eae7fa0ca897efd86ef64cd6578bc51a7.